app/src/ereferrals/java/org/eyeseetea/malariacare/data/sync/importer/metadata/configuration/MetadataConfigurationDBImporter.java

```diff
@@ -56,9 +56,8 @@ public MetadataConfigurationDBImporter(
     }
 
     public void importMetadata(Program program) throws Exception {
-        if (!needToDownloadMetadata) {
-            throw new IllegalStateException("Has to call hasToUpdateMetadata before this method.");
-        }
+        fetchContriesVersionsIfRequired();
```

Comment:
I don't understand this change in this issue, It's not related to refactoring anemic entities

Reply:
Sorry for that, I took the advantage of this refactoring to break the order calling dependency between hasToUpdateMetadata and importMetadata methods.

Before this change, you were force to call hasToUpdateMetadata method before importMetadata method or you would got an exception, just like the text reads.
```
  throw new IllegalStateException("Has to call hasToUpdateMetadata before this method.");
```